terraform/SLI_history_dataset.tf

```diff
@@ -26,7 +26,16 @@ resource "google_bigquery_table" "SLI_backup_creation_latency_view" {
   table_id = "SLI_backup_creation_latency_view"
 
   view {
-    query = "SELECT * FROM [${local.SLI_views_destination_project}:${var.SLI_history_dataset}.SLI_backup_creation_latency]"
+ query = <<EOF
+          #legacySQL
+          SELECT projectId, datasetId, tableId, partitionId, creationTime, lastModifiedTime, backupCreated, backupLastModified, xDays
```

Comment:
I would add snapshotTime in output, so that we know from which SLI run row it is

Reply:
Done